src/components/editions/article/index.tsx

```diff
@@ -95,7 +95,7 @@ export const galleryWrapperStyles = css`
 		border-right: 1px solid ${neutral[100]};
 	}
 
-	${from.wide} {
+	${from.desktop} {
```

Comment:
Not too sure we should rename the code to match the storybook viewports. We generally use `from.wide` much more in the codebase, and maybe we should get rid of `from.desktop` altogether to be more semantically correct as both apps-rendering and editions-rendering are not meant for desktop anyway? What do you think?

Reply:
@faresite  sorry could have explained my thinking here a bit better!

Basically when we use `${from.wide}` it's _so_ wide that we don't see the changes from it within any of our storybook views (obviously).

Because the Figma designs are split along the lines `tablet-potrait` and `tablet-landscape` with different layouts, if we don't use the `${from.desktop}` breakpoint we have no way of distinguishing the layouts in the code in such a way that reflects this.  Or to put it another way, `${from.desktop}` === `tablet-landscape`. 

Also, I think for consistency across the Guardian, its good to display all the Gu breakpoints in Storybook and in order for that to work we need to swap `${from.wide}` for `${from.desktop}`. 

Does that make sense to you?

Reply:
Thanks @joecowton1 , that does make sense 👍